AppTaskQueue/appscale/taskqueue/queue.py

```diff
@@ -125,6 +125,7 @@ def __init__(self, queue_info, app):
         self.task_retry_limit = retry_params['task_retry_limit']
 
     self.validate_config()
+    self.prepared_statements = {}
```

Comment:
@cdonati , do you mean it should be collected by garbage collector? Or how it explains that there is no risk of memory leak? What I see is that this dict is only extended through the time. Every time new statement is prepared it gets new item. And as this dict is referenced by Queue object and since the dict holds references to prepared statements, it seems that none of them will be collected by garbage collector.